src/nft/pages/asset/Asset.tsx

```diff
@@ -49,6 +55,28 @@ const AudioPlayer = ({
   )
 }
 
+const formatter = Intl.DateTimeFormat('en-GB', { dateStyle: 'full', timeStyle: 'short' })
+
+const CountdownTimer = ({ sellOrder }: { sellOrder: SellOrder }) => {
+  const { date, expires } = useMemo(() => {
+    const date = new Date(sellOrder.orderClosingDate)
+    return {
+      date,
+      expires: formatter.format(date),
+    }
+  }, [sellOrder])
+  const [days, hours, minutes, seconds] = useTimeout(date)
+
+  return (
+    <MouseoverTooltip text={<Box fontSize="12">Expires {expires}</Box>}>
+      <Box as="span" fontWeight="normal" className={caption} color="darkGray">
+        Expires: {days !== 0 ? `${days} days` : ''} {hours !== 0 ? `${hours} hours` : ''} {minutes} minutes {seconds}{' '}
+        seconds
+      </Box>
+    </MouseoverTooltip>
+  )
+}
+
 const AssetView = ({
```

Comment:
The back arrow on the Details page does not correctly link back to the collection page for an asset. Be sure to use a router Link component to handle the `#`

Reply:
The copy/share icon also does not have the correct url